update_copyrights

```diff
@@ -66,6 +66,8 @@ with contextlib.suppress(FileNotFoundError):
     os.remove(options.output)
 
 csv_data = {}
+# Too many or too few fields
+malformed = []
 added = []
 changed = []
 # Already mentioned in the CSV file, but lacking something in either the license or author fields
```

Comment:
I had the same thought but wanted to keep the variables declared in order of use.  Adding comments to `added` and `changed` is a good idea.  Another option is to just put the comment for `malformed` at the end of the declaration line, since it's short anyway, but that breaks the existing style.